src/runtime/recipe/handle.ts

```diff
@@ -172,6 +173,8 @@ export class Handle implements Comparable<Handle> {
     }
     this._fate = fate;
   }
+  get capabilities() { return this._capabilities; }
```

Comment:
Can we just use `capabilities` instead of `_capabilities` and avoid the getter and setter?

I don't think the extra complexity gives us any extra power.

Reply:
agreed, done